docs/generate_markdown.bash

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+###############################################################################
+# Copyright (c) 2018 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+SCRIPT_PATH=$(cd $(dirname ${0}); pwd -P)
+
+cd ${SCRIPT_PATH}/..
+
+rm -rf build_doxygen_xml
+rm -rf docs/api
+rm -f docs/overview.md
+rm -f docs/contributing/code_of_conduct.md
+rm -f docs/releases/changelog.md
+
+mkdir docs/api
+
+# We need to copy these files into the `docs/` root because Jekyll doesn't let
+# you include content outside of its root.
+cp README.md docs/overview.md
+cp CODE_OF_CONDUCT.md docs/contributing/code_of_conduct.md
+cp CHANGELOG.md docs/releases/changelog.md
```

Comment:
I get an error that `docs/releases/` does not exist. Add a `mkdir docs/releases`.

Reply:
Fixed.